CONFLICTS.md

```diff
@@ -13,14 +13,10 @@ references related issues.
   This version is missing the service alias `validator.expression`
   which causes ValidatorException exception to be thrown when using `Expression` constraint. 
 
-- `doctrine/orm:2.15.2`:
+- `doctrine/orm:>= 2.16.0`
 
-  This version introduced a bug, which causes the `ForeignKeyConstraintViolationException` exception to not be thrown when trying to delete a resource with a foreign key constraint.
-  References: https://github.com/doctrine/orm/issues/10752
-
-- `doctrine/orm:>=2.15.3`
-
-  This version introduced a bug, causing the bulk editing not to work properly. When deleting two items at once, the second one is deleted and re-added to the database.
+  This version makes Sylius Fixtures loading fail on the product review fixtures.
+  References: https://github.com/doctrine/orm/issues/10877
```

Comment:
```suggestion
  References: https://github.com/doctrine/orm/issues/10869
```

The issue you linked was closed out as a duplicate referencing this one instead

Reply:
It has a reference to the first issue, so I believe there's no big difference to which one we refer.